frontend/vue/components/Statevector/StatevectorBinaryEncoding.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <StatevectorBaseWidget
+    :number-of-elements="4"
+    :amplitude-disk-data="amplitudeDiskData"
+  >
+    <template #default>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Convallis vitae elit fermentum sit. Pellentesque congue risus sed condimentum viverra dolor. Sapien nunc vulputate posuere vel.
```

Comment:
similar question - should we wrap this in a `<p>`?

Reply:
I think this makes sense here :D